cumulusci/cumulusci.yml

```diff
@@ -8,8 +8,7 @@ tasks:
         class_path: cumulusci.tasks.salesforce.ActivateFlowProcesses.ActivateFlowProcesses
         options:
             developer_names:
-                - Auto_Populate_Date_And_Name_On_Program_Engagement
-                - ape
+                -
```

Comment:
Just leave this option out of cumulusci.yml; it doesn't make sense to set it here unless there really is a default that makes sense for all projects.

Reply:
@davisagli  I wanted to chat with you about the project scopings and where to put the options and when to document/how to handle if its project specific.

Reply:
the flow names for this task will be project-specific, so they should go in the project's cumulusci.yml like this:

    tasks:
        activate_flow:
            options:
                developer_names:
                    - Auto_Populate_Date_And_Name_On_Program_Engagement

This gets merged with the global config from CumulusCI's cumulusci.yml, so it'll inherit the task description and class_path